spec/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.new(
+      title: 'My first post',
+      text: 'Hello world!',
+      comments_counter: 5,
+      likes_counter: 8
+    )
+  end
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should be maximum 250 characters' do
+    subject.title = 'a' * 251
+    expect(subject).to_not be_valid
+  end
+
+  it 'should diplay most recent comments' do
+    subject.comments_counter = 5
+    expect(subject.recent_comments).to eq(subject.comments.last(5))
+  end
+
+  it 'should display likes_counter' do
+    subject.likes_counter = 8
+    expect(subject.likes_counter).to eq(8)
+  end
+end
```

Comment:
- [ ] @asifaminisonline great work implementing the changes requested but while changing your post_spec.rb file you removed the test for the comment counter kindly add one